Rubberduck.Parsing/Inspections/Abstract/IInspectionResult.cs

```diff
@@ -8,6 +8,9 @@ namespace Rubberduck.Parsing.Inspections.Abstract
     public interface IInspectionResult : IComparable<IInspectionResult>, IComparable
     {
         IEnumerable<IQuickFix> QuickFixes { get; }
+        bool HasQuickFixes { get; }
```

Comment:
Not sure what you added this for, but I'm going to kill it when I make the quick fixes specify which inspections they can fix and take an inspection result for the specific details.